EIPS/eip-draft_tx_exp.md

```diff
@@ -0,0 +1,86 @@
+---
+eip: <to be assigned>
+title: Expirable Trainsaction 
+description: This EIP adds a new transaction type of that includes expiration with a blocknum 
+author: Zainan Victor Zhou <zzn@zzn.im> (@xinbenlv), Nick Johnson (@Arachnid)
```

Comment:
Email address are temp sometimes. I wonder if ENS shall be allowed / recommended to use here?

Reply:
The primary purpose of the author line is so that the author can be reached to communicate with.  There is no standard way to initiate communication with someone via ENS at the moment, so I don't think ENS is a good fit (at least not today).

Reply:
DONE, removed email and use github id